setlexsem/__init__.py

```diff
@@ -1,20 +0,0 @@
-from .generate.prompt import (
```

Comment:
Why did these imports go away? I'm not opposed to a little tightening of the namespace, but am curious about your reasoning.

Reply:
Just cleaning up the repo, because we had it in some folders and not others. Installing the repo with `pip install -e .` will take care of these.

Reply:
I don't understand your response, but let's move on.